big_tests/tests/graphql_muc_light_SUITE.erl

```diff
@@ -537,6 +580,229 @@ user_blocking_list_story(Config, Alice, Bob) ->
                     <<"entity">> => RoomBin}],
                  get_ok_value(?GET_BLOCKING_LIST_PATH, Res5)).
 
+%% Domain admin test cases
+
+domain_admin_create_room_no_permission(Config) ->
+    escalus:fresh_story_with_config(Config, [{alice_bis, 1}],
+                                    fun domain_admin_create_room_no_permission_story/2).
+
+domain_admin_create_room_no_permission_story(Config, AliceBis) ->
+    AliceBisBin = escalus_client:short_jid(AliceBis),
+    InvalidUser = make_bare_jid(?UNKNOWN, ?UNKNOWN_DOMAIN),
+    MucServer = ?config(muc_light_host, Config),
+    Name = <<"first room">>,
+    Subject = <<"testing">>,
+    % Try with a non-existent domain
+    Res = create_room(MucServer, Name, InvalidUser, Subject, null, Config),
+    get_unauthorized(Res),
+    % Try with an external domain
+    Res2 = create_room(MucServer, Name, AliceBisBin, Subject, null, Config),
+    get_unauthorized(Res2).
+
+domain_admin_create_identified_room_no_permission(Config) ->
+    escalus:fresh_story_with_config(Config, [{alice_bis, 1}],
+                                    fun admin_create_identified_room_no_permission_story/2).
+
+admin_create_identified_room_no_permission_story(Config, AliceBis) ->
+    AliceBisBin = escalus_client:short_jid(AliceBis),
+    InvalidUser = make_bare_jid(?UNKNOWN, ?UNKNOWN_DOMAIN),
+    MucServer = ?config(muc_light_host, Config),
+    Name = <<"first room">>,
+    Subject = <<"testing">>,
+    SchemaEndpoint = atom_to_binary(?config(schema_endpoint, Config)),
+    Protocol = atom_to_binary(?config(protocol, Config)),
+    Id = <<"my_room_", SchemaEndpoint/binary, "_", Protocol/binary>>,
+    % Try with a non-existent domain
+    Res = create_room(MucServer, Name, InvalidUser, Subject, Id, Config),
+    get_unauthorized(Res),
+    % Try with an external domain
+    Res2 = create_room(MucServer, Name, AliceBisBin, Subject, Id, Config),
+    get_unauthorized(Res2).
+
+domain_admin_change_room_config_no_permission(Config) ->
+    escalus:fresh_story_with_config(Config, [{alice, 1}, {bob, 1}],
+                                    fun domain_admin_change_room_config_no_permission_story/3).
+
+domain_admin_change_room_config_no_permission_story(Config, Alice, Bob) ->
+    AliceBin = escalus_client:short_jid(Alice),
+    BobBin = escalus_client:short_jid(Bob),
+    MUCServer = ?config(muc_light_host, Config),
+    ExternalDomain = domain_helper:secondary_domain(),
+    RoomName = <<"first room">>,
+    {ok, #{jid := #jid{luser = RoomID} = RoomJID}} =
+        create_room(MUCServer, RoomName, <<"subject">>, AliceBin),
+    {ok, _} = invite_user(RoomJID, AliceBin, BobBin),
+    % Try to change the config with a non-existent domain
+    Res = change_room_configuration(
```

Comment:
I have to note that both jid's in this test point to non-existing rooms at domains that are not MUC-Light domains. It would be better to test an existing room at a different domain.